lib/system.nim

```diff
@@ -2134,26 +2134,25 @@ const
 import system/dollars
 export dollars
 
-proc delete*[T](x: var seq[T], i: Natural) {.noSideEffect.} =
+proc delete*[T](x: var seq[T], i: Natural) {.noSideEffect, deprecated: "use sequtils.delete(x, i..i) instead".} =
```

Comment:
```suggestion
proc delete*[T](x: var seq[T], i: Natural) {.noSideEffect, deprecated: "use `sequtils.delete(x, i..i)` instead".} =
```

Reply:
I dont know, other `"use X instead"` do not have pseudocode for the arguments anyway, would be better to be consistent. 🤔

Reply:
Eh, most deprecations I found used backticks for the replacement function/expression.

Reply:
> I dont know, other "use X instead" do not have pseudocode for the arguments anyway, would be better to be consistent.

Well we cannot all be so smart and remove useful information for "consistency".